bindings/hnsw_bindings.cpp

```diff
@@ -125,10 +138,15 @@ template <typename dist_t, typename data_t = float> class Index {
   }
 
   void init_new_index(const size_t maxElements, const size_t M,
-                      const size_t efConstruction, const size_t random_seed) {
+                      const size_t efConstruction, const size_t random_seed,
+                      py::object using_pq) {
     if (appr_alg) {
       throw new std::runtime_error("The index is already initiated.");
     }
+    if (!using_pq.is_none()) {
+      loadPQ(using_pq);
+      normalize = false; // PQ distance metrics default l2
```

Comment:
let's raise an issue to explain why `normalize` must be false. Can we overcome this limitation? We need a discussion about it.